ganttproject/src/net/sourceforge/ganttproject/gui/options/CSVSettingsPanel.java

```diff
@@ -70,7 +70,7 @@ public CSVSettingsPanel(UIFacade uiFacade, CSVOptions csvOptions) {
     applyComponentOrientation(language.getComponentOrientation());
   }
 
-  JComponent createSeparatorSettingsPanel() {
+  JComponent createSeparatorSettingsPanel(String id) {
```

Comment:
I don't think that we want to parameterize it with id. This code knows what it builds, uses i18n directly and it is okay to let this code choose the appropriate id by itself